src/outputs.ts

```diff
@@ -61,7 +61,7 @@ export function validateOutput(network, output, inputsTotalSats?) {
 /**
  * Lowest acceptable output amount in Satoshis.
  */
-const DUST_LIMIT_SATS = new BigNumber(546);
+const DUST_LIMIT_SATS = new BigNumber(546).toString();
```

Comment:
Couldn’t this just be 
```suggestion
const DUST_LIMIT_SATS = “546”
```

Reply:
Yes. I don't think the difference matters that much. I think there is a difference in that the first is like a casting of an int and the second is purely a string which may not have started as an int.